dejavu/__init__.py

```
@@ -111,6 +132,61 @@ def fingerprint_file(self, filepath, song_name=None):
             self.db.set_song_fingerprinted(sid)
             self.get_fingerprinted_songs()
 
+    def fingerprint_with_duration_check(self, input_file, minutes=5, song_name=None, processes=None):
+        duration = get_duration(input_file)
+        split_length =  minutes * 60
+        if duration < split_length:
+            return self.fingerprint_file(input_file)
+        songname, extension = os.path.splitext(os.path.basename(input_file))
+        song_name = song_name or songname
+        # don't refingerprint already fingerprinted files
+        if song_name in self.songnames_set:
+            print "%s already fingerprinted, continuing..." % song_name
+            return
+        file_directory = os.path.dirname(input_file)
+        output_split_path = os.path.join(file_directory, self.SPLIT_DIR)
+        try:
+            os.mkdir(output_split_path)
+        except WindowsError:
+            pass
+        output_path = os.path.join(output_split_path, song_name)
+        try:
+            os.mkdir(output_path)
+        except WindowsError:
+            pass
```

Why are you ignoring the error here with just a `pass`?